### PR TITLE
Copy the Django Code of Conduct

### PR DESCRIPTION
**Pull request by danfinlay**
Why not begin with this as a starting point? Thoughts?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,35 @@
...
 
-― Kurt Vonnegut
+Diversity is one of our huge strengths, but it can also lead to communication issues and unhappiness. To that end, we have a few ground rules that we ask people to adhere to. This code applies equally to founders, mentors and those seeking help and guidance.
+
+This isn’t an exhaustive list of things that you can’t do. Rather, take it in the spirit in which it’s intended - a guide to make it easier to enrich all of us and the technical communities in which we participate.
+
+This code of conduct applies to all spaces managed by the MetaMask project or MetaMask Software Foundation. This includes IRC, the mailing lists, the issue tracker, DSF events, and any other forums created by the project team which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+
+If you believe someone is violating the code of conduct, we ask that you report it by emailing conduct@djangoproject.com. For more details please see our Reporting Guidelines
```
Comment (brunobar79) [inline README.md:15]: change the email

File: README.md [hunk 2]
```diff
@@ -1,10 +1,35 @@
...
+Like the technical community as a whole, the MetaMask team and community is made up of a mixture of professionals and volunteers from all over the world, working on every aspect of the mission - including mentorship, teaching, and connecting people.
 
-― Kurt Vonnegut
+Diversity is one of our huge strengths, but it can also lead to communication issues and unhappiness. To that end, we have a few ground rules that we ask people to adhere to. This code applies equally to founders, mentors and those seeking help and guidance.
+
+This isn’t an exhaustive list of things that you can’t do. Rather, take it in the spirit in which it’s intended - a guide to make it easier to enrich all of us and the technical communities in which we participate.
+
+This code of conduct applies to all spaces managed by the MetaMask project or MetaMask Software Foundation. This includes IRC, the mailing lists, the issue tracker, DSF events, and any other forums created by the project team which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
```
Comment (brunobar79) [inline README.md:13]: Things to change:
- Metamask Software Foundation (not 100% sure about this one)
- DSF events

On a separate note, should we mention ConsenSys here at all?

File: README.md (hunk 1 above)
Comment (brunobar79) [reply README.md:15]: Maybe we could set up an anonymous google form for this purpose and send a copy to all

File: README.md [hunk 3]
```diff
@@ -1,10 +1,35 @@
...
+        Personal insults, especially those using racist or sexist terms.
+        Unwelcome sexual attention.
+        Advocating for, or encouraging, any of the above behavior.
+        Repeated harassment of others. In general, if someone asks you to stop, then stop.
+    When we disagree, try to understand why. Disagreements, both social and technical, happen all the time and MetaMask is no exception. It is important that we resolve disagreements and differing views constructively. Remember that we’re different. The strength of MetaMask comes from its varied community, people from a wide range of backgrounds. Different people have different perspectives on issues. Being unable to understand why someone holds a viewpoint doesn’t mean that they’re wrong. Don’t forget that it is human to err and blaming each other doesn’t get us anywhere. Instead, focus on helping to resolve issues and learning from mistakes.
+
+Original text courtesy of the Speak Up! project.
+Questions?
 
+If you have questions, please see the FAQ. If that doesn't answer your questions, feel free to contact us.
```
Comment (brunobar79) [inline README.md:35]: Not sure what to do with the FAQ here.

**Review by brunobar79 — APPROVED** [review]
I think this is great! 

I'd love to see each point "Be something" in bold.

File: README.md (hunk 1 above)
Comment (brunobar79) [reply README.md:15]: Also not sure about our reporting guidelines. Should we create ours or maybe fork it too?

**Comment by whymarrh** [comment]
Replaced with https://github.com/MetaMask/.github/pull/3, which will properly propagate the file to all the repos in the org.

**Pull request closed without merging**